### ISSUE TITLE
Why is this package not on pypi or conda anymore?

### ISSUE THREAD

**Issue by Erotemic** [dgketchum/Landsat578#39]
Is it just a maintenance thing? I have extensive experience automatically publishing to pypi using CI or via a 1-command script. Let me know if that's the issue. 

If its not, I'm curious why its only hosted on gitlab now.

**Comment by dgketchum** [comment]
Hi. Unfortunately I no longer use this project and have not spent time maintaining it. For this reason, if I ever edit the project I just push to the github master. Is it hosted on gitlab?

Cheers,

**Comment by Erotemic** [comment]
Makes sense. Thanks for responding. When I said gitlab it was a typo. I meant github.

Thanks for starting the project in the first place, while the code didn't run on my machine (pandas import paths changed), it served as a useful reference! I trying to programatically download a landsat example image, and I couldn't figure out how to do it before I saw this repo. Your work made my life easier, and I appreciate it :smile:
